README.md

```diff
@@ -158,36 +158,16 @@ apt-get install -y \
 
 ### Get Source Code
 
-As with other Go projects, CRI-O must be cloned into a directory structure like:
-
-```
-GOPATH
-└── src
-    └── github.com
-        └── kubernetes-incubator
-            └── cri-o
-```
-
-First, configure a `GOPATH` (if you are using go1.8 or later, this defaults to `~/go`).
-
-```bash
-export GOPATH=~/go
-mkdir -p $GOPATH
-```
-
-Next, clone the source code using:
+Clone the source code using:
 
 ```bash
-mkdir -p $GOPATH/src/github.com/kubernetes-incubator
-cd $_ # or cd $GOPATH/src/github.com/kubernetes-incubator
 git clone https://github.com/kubernetes-incubator/cri-o # or your fork
 cd cri-o
 ```
 
 ### Build
 
 ```bash
-make install.tools
```

Comment:
If you don't do the make install.tools, I don't think go-md2man, gometalinter  and git-validation will be in play.

Reply:
> If you don't do the make install.tools, I don't think go-md2man, gometalinter and git-validation will be in play.

You may have them already.  We could potentially detect that and have `.install.gitvalidation` only act if `git-validation` wasn't in your `PATH`, etc.  And we need `go-md2man` to build `docs`, which is a prereq of the default `all` target.  So I'm in favor of keeping this line as it was.

Reply:
For the simple checkout + `make` + `make install` these do not seem to be needed.

For workflows / development work where they are needed, the `make install.tools` can of course be mentioned in that workflow description.

Reply:
Fair enough. But if something is prerequisite for something else, why not just mark it as such for the target in the Makefile? The fewer steps people have to make manually (or put into their build scripts), the fewer things can change and break in the future.

Reply:
> But if something is prerequisite for something else, why not just mark it as such for the target in the Makefile?

That would be best; patches welcome ;).